index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <title>Njoroge</title>
+</head>
+
+<body>
+    <div class="container">
+        <header class="top-header">
+            <a href="https://github.com/francis450" class="header-item logo">My logo</a>
+            <nav class="header-item"><img src="img/Union.svg" alt=""></nav>
+        </header>
```

Comment:
- [ ] Good job adding this nav-bar :heavy_check_mark: , I noticed that your design has a bit margin-top which is big that the one used in the Figma template, kindly make sure you stick as much as possible to the Figma design, that means make sure attributes such as `margin` `padding` `fonts` are at least the same :100: 

| ### Your design| ### Figma Design |
| :---: | :---: |
| ![Screenshot from 2023-01-13 18-29-16 (1)](https://user-images.githubusercontent.com/80458199/212371421-ca0973a0-d0e3-47a0-a4cf-422a57df94f3.png) |  ![Screenshot from 2023-01-13 18-35-04](https://user-images.githubusercontent.com/80458199/212371617-39ac2e76-67ee-4f98-9557-1aa2c439274d.png) |